### PR TITLE
feat: show Neutrino syncing status

### LINKED ISSUE #778: `status` should return sync % of Neutrino
Currently, `status` is returning an rpc error if lnd's neutrino is still syncing. It should return the current sync %. This is needed for XUD Explorer to show the sync progress in the `status` field of lnd's instead of this ugly rpc error:

image

### PR DESCRIPTION
**Pull request by reliveyy**
This PR adds LND Neutrino syncing details in status API. Resolves https://github.com/ExchangeUnion/xud-docker/issues/778

### How to test?

```
bash xud.sh # choose testnet
scripts/run.sh testnet
curl -k https://localhost:8080/api/v1/status/lndbtc
```

### CODE REVIEW

File: service/lnd/lnd.go [hunk 1]
```diff
@@ -201,6 +248,85 @@ func (t *LndService) getCurrentHeight() (uint32, error) {
...
+	t.GetLogger().Debug("[watch] Neutrino syncing")
+	c, err := t.GetContainer()
+	for err != nil {
+		t.GetLogger().Debug("[watch] Waiting for Docker container to be created")
+		time.Sleep(1 * time.Second)
+	}
+	t.GetLogger().Debug("[watch] Got container")
+	startedAt := c.Unwrap().State.StartedAt
+	t.GetLogger().Debugf("[watch] startedAt=%s", startedAt)
+	logs, err := t.FollowLogs("1h", "")
```
Comment (reliveyy) [inline service/lnd/lnd.go:261]: "1h" needs to be fixed to startedAt

File: service/single_container_service.go [hunk 2]
```diff
@@ -85,7 +85,16 @@ func (t *SingleContainerService) GetLogs(since string, tail string) (<-chan stri
 	if err != nil {
 		return nil, err
 	}
-	return c.GetLogs(since, tail)
+	return c.GetLogs(since, tail, false)
+}
+
+// GetContainerLog is a shortcut function
+func (t *SingleContainerService) FollowLogs(since string, tail string) (<-chan string, error) {
```
Comment (reliveyy) [inline service/single_container_service.go:92]: FollowLogs should return a close handle function to terminate following

File: service/lnd/lnd.go [hunk 3]
```diff
@@ -201,6 +248,85 @@ func (t *LndService) getCurrentHeight() (uint32, error) {
...
+	}
+
+	t.GetLogger().Debug("[watch] Done")
+
+	return nil
+}
+
+func (t *LndService) Neutrino() bool {
+	// TODO get lnd backend type
+	return true
```
Comment (reliveyy) [inline service/lnd/lnd.go:308]: Should return true/false based on backend type.

**Review by krrprr — APPROVED** [review]
LGTM :+1:

**Review by raladev — APPROVED** [review]

**Pull request merged**